src/ggml-cuda.cu

```diff
@@ -7634,7 +7643,7 @@ static void ggml_cuda_op_mul_mat_cublas(
             cublasSgemm(g_cublas_handles[id], CUBLAS_OP_T, CUBLAS_OP_N,
                     row_diff, src1_ncols, ne10,
                     &alpha, src0_ddf_i, ne00,
-                            src1_ddf_i, ne10,
+                            src1_ddf1_i, ne10,
                     &beta,  dst_dd_i,   ldc));
```

Comment:
```suggestion
CUBLAS_CHECK(
            cublasGemmEx(g_cublas_handles[id], CUBLAS_OP_T, CUBLAS_OP_N,
                    row_diff, src1_ncols, ne10,
                    &alpha, src0_ddf_i, CUDA_R_32F, ne00,
                                src1_ddf1_i, CUDA_R_32F, ne10,
                    &beta,   dst_ddf1, CUDA_R_32F, ldc,
                    CUBLAS_COMPUTE_32F,
                    CUBLAS_GEMM_DEFAULT_TENSOR_OP));
```

Reply:
@Green-Sky Can you test the modifications I suggested above? I am not a professional CUDA developer, but after my testing, such modifications will greatly reduce the probability of bad images. However, I am not sure whether there is a compatibility problem with cublasGemmEx.

Reply:
where does the `dst_ddf1` come frome?
```
/build/xxx-source/ggml/src/ggml-cuda.cu(7456): error: identifier "dst_ddf1" is undefined
```

Reply:
Sorry it‘s `dst_dd_i `

Reply:
I don't think this is necessary.

Reply:
i cant really see any significant correlation to the issues i observe. Since they look like synchronization issues, a slightly different invocation can accidentally make them go away or less likely.

Reply:
You can use stable-diffusion.cpp
 Will `save_tensor_to_file` in [sample]( https://github.com/FSSRepo/stable-diffusion.cpp/blob/421e39b0f0aa30dcc35151f69028a7b224943196/stable-diffusion.cpp#L6258) export the tensor? The export fails and succeeds in the case of the same seed. Yes, then export the failed and successful ones at [decode](https://github.com/FSSRepo/stable-diffusion.cpp/blob/421e39b0f0aa30dcc35151f69028a7b224943196/stable-diffusion.cpp#L6272).

Reply:
When I tested, the tensor in the sample and the failed tensor were equal.

Reply:
But in the case of decoding, they have a large deviation. After I added the correction of cublasGemmEx, the deviation became smaller, but I am more confused as to why such a deviation occurs.

Reply:
Thanks, @slaren, for the style fix and other updates. 
@Cyberhan123, I'll leave it to @slaren and others to decide whether replacing `cublasSgemm` with `cublasGemmEx` is a good idea. I am not an export on CUDA/CUBlas.